app/src/js/controllers/CollectionsController.js

```diff
@@ -20,5 +22,20 @@ angular.module('media_manager').controller('CollectionsController', [
         lc.deleteCollectionModal = CollectionBehavior.deleteCollectionModal;
         lc.actuallyDeleteCollection = CollectionBehavior.actuallyDeleteCollection;
         lc.isLoadingCollections = CourseCache.isLoadingCollections;
+
+        lc.dragControlListeners = {
+          orderChanged: function(event){
+
+            lc.collections.forEach(function(item, index, arr){
+              var newsort = index + 1;
+              if(item.sort_order !== newsort){
```

Comment:
This comparison is using strict equality. Are we sure that `item.sort_order` is always going to be a numeric type?